### DISCUSSION TITLE
recipes/breads/pull-apart-sour-cream-and-chive-rolls/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#842] [Comments] [answered]
# recipes/breads/pull-apart-sour-cream-and-chive-rolls/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/pull-apart-sour-cream-and-chive-rolls/

**Comment by nicholaswilde** [comment, marked as answer]
Make a half batch
Used instant yeast that clumped in water. Try adding IY to dry ingredients directly without blooming.
